test/e2e/core/popupTabNavigation.test.js

```diff
@@ -215,7 +215,7 @@ test.describe("Playwright Keyboard Navigation + Query Layer Tests" , () => {
       const nh = await page.evaluateHandle(doc => doc.shadowRoot, h);
       const rh = await page.evaluateHandle(root => root.activeElement, nh);
       const f = await (await page.evaluateHandle(elem => elem.innerText, rh)).jsonValue();
-      expect(f).toEqual("Maps4HTML");
+      expect(f).toEqual("Leaflet");
```

Comment:
Any reason why the order of the Maps4HTML and Leaflet attribution is being changed on the attribution control? I think we should keep the order it currently is in, with Maps4HTML attribution first and then Leaflet.